src/components/speakers-section/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { tw } from 'twind';
+import { loadFull } from 'tsparticles';
+import Particles from 'react-tsparticles';
+import { useEffect, useState } from 'react';
```

Comment:
It's a good practice to organize imports always importing react as first one, then other libraries, then some components of our code an then files or components those are in the same directory. Something like this: 
```suggestion
import { useEffect, useState } from 'react';
import Particles from 'react-tsparticles';
import { loadFull } from 'tsparticles';
import { tw } from 'twind';
```